journalbeat/reader/journal.go

```diff
@@ -256,6 +256,15 @@ func (r *Reader) toEvent(entry *sdjournal.JournalEntry) *beat.Event {
 		fields.Put("journald.custom", custom)
 	}
 
+	// if entry is coming from a remote journal, add_host_metadata overwrites the source hostname, so it
+	// has to be copied to a different field
+	if r.config.Remote {
+		remoteHostname, err := fields.GetValue("host.hostname")
+		if err == nil {
+			fields.Put("source.hostname", remoteHostname)
```

Comment:
Is the same thing in Filebeat called `log.source.address` ([docs](https://www.elastic.co/guide/en/beats/filebeat/current/exported-fields-log.html))? If so then I think it would be good to stay consistent.